src/index.js

```diff
@@ -46,6 +47,15 @@ function CopyWebpackPlugin(patterns = [], options = {}) {
         let contextDependencies;
         const written = {};
 
+        let context;
```

Comment:
`\n` between the variable declaration and the if block please :)

Reply:
Should I place it up with the other `let`s and `const`s as well?